vllm/config.py

```diff
@@ -55,6 +58,9 @@ class ModelConfig:
         max_context_len_to_capture: Maximum context len covered by CUDA graphs.
             When a sequence has context length larger than this, we fall back
             to eager mode.
+        serialize: In the event that serialized weights can't be found at
```

Comment:
in retrospect: adding this will make the tensorizer behavior inconsistent with the default huggingface transformers behavior, which will save the weights to the download_dir whether or not this argument is here. additionally: I don't think this belongs in the model config.